docs/usage.rst

```diff
@@ -147,6 +156,38 @@ mutate it however you want, it will only be processed upon leaving the
     meant to be used on the whole feature.  This is because Django's
     ``LiveServerTestCase`` resets the test database after each scenario.
 
+
+Fixtures using a decorator
+**************************
+
+You can define `django fixtures`_ or a path to a callable for any step, using
```

Comment:
I think, "Django" needs to be written with a capital initial letter. -- Unless you're talking about [django-fixtures](https://pypi.python.org/pypi/django-fixtures). :smiley: